block/manager.go

```diff
@@ -185,22 +201,28 @@ func (m *Manager) Start(ctx context.Context) error {
 
 	// listen to new bonded sequencers events to add them in the sequencer set
 	go uevent.MustSubscribe(ctx, m.Pubsub, "newBondedSequencer", settlement.EventQueryNewBondedSequencer, m.UpdateSequencerSet, m.logger)
+	uerrors.ErrGroupGoLog(eg, m.logger, func() error {
+		return m.SyncLoop(ctx)
+	})
+
+	err = m.syncFromSettlement()
+	if err != nil {
+		return fmt.Errorf("sync block manager from settlement: %w", err)
+	}
```

Comment:
it's very non-obvious why there are two different sync routines happening here
could you add some comment or maybe rename something/

Reply:
naming issue. i renamed it to make it clear. syncFromSettlement only fetches last state from hub, but it does not sync.